components/src/assets/localization/en/protocol_command_text.json

```
@@ -54,7 +54,7 @@
   "move_to_addressable_area_drop_tip": "Moving to {{addressable_area}}",
   "move_to_coordinates": "Moving to (X: {{x}}, Y: {{y}}, Z: {{z}})",
   "move_to_slot": "Moving to Slot {{slot_name}}",
-  "move_to_well": "Moving to well {{well_name}} of {{labware}} in {{labware_location}}",
+  "move_to_well": "Moving to well {{wellName}} of {{labware}} with x,y,z offset {{xOffset}}, {{yOffset}}, {{zOffset}} relative to {{positionRelative}} in {{displayLocation}}",
```

Maybe this is a bit more clear but I am no language expert. @ecormany can probably suggest a better sentence structure.
I am cool with merging this as is since it's still better than the text we currently have.

```suggestion
  "move_to_well": "Moving to x,y,z offset {{xOffset}}, {{yOffset}}, {{zOffset}} relative to {{positionRelative}} of well {{wellName}} of {{labware}} in {{displayLocation}}",
```

---

Haha! I was a minute late in posting my review. I like your suggestion @ecormany.